connectors/es/sink.py

```diff
@@ -365,6 +374,13 @@ async def run(self, generator, job_type):
         except asyncio.CancelledError:
             self._logger.info("Task is canceled, stop Extractor...")
             raise
+        except asyncio.QueueFull as e:
+            self._logger.error("Sync was throttled by Elasticsearch")
+            # We clear the queue as we could not actually ingest anything.
+            # After that we indicate that we've encountered an error
+            self.queue.clear()
```

Comment:
Do you think it could be helpful to log how many documents were deleted? And maybe also log the ids. This could help, if you want to know, which documents were dropped.

Reply:
I'm not sure why - what's the benefit of logging them?

Reply:
To expand my thoughts: I'm not sure what information it will give me.

Imagine you're syncing 50.000 documents. Elasticsearch got overloaded once 4.500 documents were inserted + 500 are stuck in the queue. So I can log that we dropped 500 operations and their metadata (index + id), but I'm not sure if it helps with anything.

Reply:
I'm just thinking about support cases here, where want to know, why documents we would've expected are missing. Also we could create some telemetry on top of these log messages to get a feeling of how severe these overload errors are (which percentage of documents are dropped).

Reply:
> which percentage of documents are dropped

I don't think it's possible to make a useful metric out of it - if source has 100.000 documents, we've ingested 10.000 and dropped 500, how can this metric help? We know that we dropped 5% of whatever was ingested, but we don't know document total to tell how many were dropped. And since 90.000 documents were not ingested, this 500 is a drop in the ocean.

Maybe I misunderstand something?